monero-outreach-docs/translations/es/archive/we_accept_monero_es.md

```diff
@@ -2,15 +2,15 @@
 
 **“¿Estás cansado de implorar por adopción? ¿Harto de esperar el amanecer de Monero? Se parte del cambio que quieres ver en el ecosistema.” - 20th of MAy, 2020**
 
-Nos gustaría invitarte a unirte a nuestros esfuerzos para llevar a cabo una buena 'campaña agobiante' a la antigua con tus comerciantes favoritos que aún no aceptan XMR. La 'campaña agobiante' más impresionante y exitosa de todos los tiempos tuvo que haber sido escrita por George Lois. En 1982, las empresas de cable no tenían interés en un canal de música las 24 horas, no creían que funcionaría. A George Lois se le ocurrió una idea y filmó una serie de comerciales en los que las estrellas de rock pedían a los espectadores que llamaran a su compañía de cable local y dijeran "¡Quiero mi MTV!" El resto es historia, [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
+Nos gustaría invitarte a unirte a nuestros esfuerzos para llevar a cabo una buena 'campaña agobiante' a la antigua con tus comerciantes favoritos que aún no aceptan XMR. La 'campaña agobiante' más impresionante y exitosa de todos los tiempos tuvo que haber sido escrita por George Lois. En 1982, las empresas de cable no tenían interés en un canal de música las 24 horas, no creían que funcionaría. A George Lois se le ocurrió una idea y filmó una serie de comerciales en los que las estrellas de rock pedían a los espectadores que llamaran a su operadora de televisión local" y dijeran "¡Quiero mi MTV!" El resto es historia, [youtu.be/vAyOcWpm3QU](https://youtu.be/vAyOcWpm3QU?t=71).
 
-La misión, si decides aceptarlo, es comunicarte con tu comerciante favorito y preguntarle si ha considerado aceptar Monero, porque ese sería su método de pago preferido. Puede que digan que sí, que digan que no, pero al menos les has hecho pensar en ello. No será la última vez que escuchen sobre Monero, pero podría ser la primera.
+La misión, si decides aceptarlo, es comunicarte con tu comerciante favorito y preguntarle si ha considerado aceptar Monero, porque ese sería tu método de pago preferido. Puede que digan que sí, que digan que no, pero al menos les has hecho pensar en ello. No será la última vez que escuchen sobre Monero, pero podría ser la primera.
```

Comment:
I have a suggestion. 

"La misión, si decides **acpetarla**,..."

Reply:
ok

Reply:
Done, It was fixed